include/xten/Conversion/ATenToXTen.td

```diff
@@ -29,8 +29,11 @@ def CheckFalse : Constraint<
 
 // Conversions
 
-def : Pat<(Torch_AtenConv2dOp $a,$b,$c,$d,$e,$f,$g),
-          (XTen_Conv2dOp  $a,$b,$c,$d,$e,$f,$g)>;
+// def : Pat<(Torch_AtenConv2dOp $a,$b,$c,$d,$e,$f,$g),
```

Comment:
Should we not remove that ? Or if we decide to keep it, I'd rather have it uncommented